service/worker/replicator/replicationTask.go

```diff
@@ -128,6 +131,7 @@ func newActivityReplicationTask(
 			timeSource:    timeSource,
 			historyClient: historyClient,
 			metricsClient: metricsClient,
+			domainTag:     common.GetDomainTagByName(domainCache.GetDomainName(attr.GetDomainId())),
```

Comment:
Separate it to two lines?
domainName := domainCache.GetDomainName(attr.GetDomainId())
common.GetDomainTagByName(domainName)